### PR TITLE
Modernize

### PR DESCRIPTION
**Pull request by gijzelaerr**

### CODE REVIEW

**Comment by gijzelaerr** [comment]
depends on https://github.com/ska-sa/kittens/pull/4 and https://github.com/ska-sa/tigger-lsm/pull/4

File: TigGUI/tigger [hunk 1]
```diff
@@ -91,7 +92,7 @@ if __name__ == "__main__":
     import TigGUI.MainWindow
     dprint(1,"imported TigGUI.MainWindow")
     import TigGUI.Tools
-    from TigGUI.Tools import export_karma,add_brick,make_brick,restore_image
+
```
Comment (o-smirnov) [inline TigGUI/tigger]: As far as I remember, these things dynamically insert themselves into the menus when imported. So you may have just killed them here...

**Comment by o-smirnov** [comment]
OK, I've gone through the commits one by one, and apart from https://github.com/ska-sa/tigger/pull/95#pullrequestreview-117555030, I have nothing meaningful to say.

The monster reformatting ones are of course impossible to really review. We just have to trust in the tools.

**Comment by o-smirnov** [comment]
OK, so here's an issue. The ``tigger-lsm`` package also uses Kittens (in particular, Kittens.utils and Kittens.config).

File: TigGUI/tigger (hunk 1 above)
Comment (gijzelaerr) [reply TigGUI/tigger]: i have no words for this. i'll restore it.

Comment (gijzelaerr) [reply TigGUI/tigger]: Wait i see no difference in tigger If i comment it or uncomment it... what should this do?

**Comment by o-smirnov** [comment]
@gijzelaerr can you check my commit here? https://github.com/ska-sa/tigger/pull/95/commits/a1e12cdf53ed70957e2f7ac0d81c9724c17f9933

I'm kind of monkeying around blindly here, as I don't quite know how Unicode works.

**Comment by o-smirnov** [comment]
OK why does https://github.com/ska-sa/tigger/blob/modernize/TigGUI/tigger#L114 now print a tuple.

**Pull request merged**